urumanifest/github.py

```diff
@@ -383,10 +385,17 @@ def _unpack_artifact(staging_path: Path, database: _WorkflowDatabase, rev: str,
         output_path.mkdir(parents=True, exist_ok=True)
 
         def iter_client_dir():
-            for i in filter(lambda x: not x.is_dir(), archive.infolist()):
+            for i in archive.infolist():
                 member_path = PurePosixPath(i.filename)
-                if len(member_path.parts) == 2 and member_path.parts[0] == "client":
-                    yield i
+                if len(member_path.parts) >=  2 and member_path.parts[0] == "client":
+                    is_mac_app_bundle = member_path.parts[1].endswith(".app")
+                    if not i.is_dir() and len(member_path.parts) ==  2:
+                        yield (Path(i.filename).name, i, Path(i.filename).name)
```

Comment:
I would prefer to see this return a `NamedTuple` so I can know what exactly I'm dealing with in terms of return values.